tests/test_update_image.py

```diff
@@ -0,0 +1,85 @@
+#!/usr/bin/env python3
+# SPDX-License-Identifier: GPL-2.0-only
+
+#
+# Copyright (C) 2019 Datto, Inc.
+# Additional contributions by Elastio Software, Inc are Copyright (C) 2020 Elastio Software Inc.
+#
+
+import math
+import errno
+import os
+import platform
+import unittest
+
+import elastio_snap
+import util
+from devicetestcase import DeviceTestCase
+
+
+class TestUpdateImage(DeviceTestCase):
+    def setUp(self):
+        self.snap_mount = "/mnt"
+        self.snap_device = "/dev/elastio-snap{}".format(self.minor)
+        self.snap_bkp = "./base-image.img"
```

Comment:
This image is base only once after `setup-snapshot`. Then it's base+incremental, base+2incrementals etc ) Let's call it `backup.img` or somehow else.

Reply:
Done.